TwentyOneTournament2016/TwentyOneTournament2016/PlayoffsR2.swift

```diff
@@ -9,8 +9,64 @@
 import Foundation
 
 
-class PlayoffsR2: UIViewController {
+class PlayoffsR2: UIViewController, UITableViewDataSource, UITableViewDelegate{
+    
+
+    @IBOutlet var tableView: UITableView!
+    
+    let cellId = "r2Cell"
+    
+    var midCells = [3,7,11,15]
+    
+    
+    override func viewDidLoad() {
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 16
+    }
+    
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier(cellId, forIndexPath: indexPath)
+        
+        let row = indexPath.row
+        
+        if (row + 1) % 4 == 0{
+            cell.backgroundView = UIImageView (image: UIImage (named: "r2lowerBracket"))
+            cell.backgroundColor = UIColor.redColor()
+        } else if (row + 1) % 2 == 0{
+            cell.backgroundView = UIImageView (image: UIImage (named: "r2topBracket"))
+            cell.backgroundColor = UIColor.redColor()
+        }else if (row + 1) % 2 != 0 {
+            for n in 0...3{
+                if row+1 == midCells[n]{
+                    cell.backgroundView = UIImageView (image: UIImage (named: "r2midBracket"))
+                    cell.backgroundColor = UIColor.redColor()
+                }else{
+                    cell.backgroundColor = UIColor.redColor()
+                }
+            }
+        }else{
+            cell.backgroundColor = UIColor.redColor()
+        }
+        
```

Comment:
@bbniedert this whole block of figuring out which image to place on which cell on the round 2 screen is where things are getting pretty screwed up. Any idea of a better way to do this? Specifically for lines 48-56. Those lines are determining which cell to point the center line image on. I know I'm not doing it in the best way but can't think of a better solution.